### PR TITLE
Add HHVM 4.153 LTS to Release Schedule

### PR DESCRIPTION
**Pull request by AndrewDiMola**
Add 4.153 LTS and update past end of support dates.

image

### CODE REVIEW

**Review by Atry — APPROVED** [review]

File: guides/hhvm/01-installation/06-release-schedule.md [hunk 1]
```diff
@@ -11,11 +11,12 @@ Here's a table of recent LTS releases as well as *expected* future LTS releases:
 
 | Version |    \*Release Date       |    \*End of Suppport       |
```
Comment (lexidor) [inline guides/hhvm/01-installation/06-release-schedule.md:12]:
```suggestion
| Version |    \*Release Date       |    \*End of Support       |
```

**Review by lexidor — APPROVED** [review]

File: guides/hhvm/01-installation/06-release-schedule.md (hunk 1 above)
Comment (AndrewDiMola) [reply guides/hhvm/01-installation/06-release-schedule.md]: wow, nice catch.

Comment (AndrewDiMola) [reply guides/hhvm/01-installation/06-release-schedule.md]: End of `ppp` support. End of support: today.

Comment (lexidor) [reply guides/hhvm/01-installation/06-release-schedule.md]: Ending support very very softly. 🎼

**Pull request merged**